email-templates/weekly/html.ejs

```diff
@@ -0,0 +1,579 @@
+                                <tr>
+                                  <td class="most-points-value">
+                                    <%= averagePoints %> points
+                                  </td>
+                                </tr>
+                              </tbody>
+                            </table>
+                          </td>
+
+                        </tr>
+                      </tbody>
+                    </table>
+                  <%
+                  }
+                  -%>
+
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </td>
+        </tr>
+
+        <tr>
+          <td class="centered">
+          <% 
+          if (course.engagementindex_url) { 
+          -%>
+            <div class="outbound-button">
+              <a href="<%= course.engagementindex_url %>">
+                View leaderboard
+              </a>
+            </div>
+          <% 
+          } 
+          -%>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <h4 class="section-header">
+              Your activity
+            </h4>
+
+            <table cellpadding="0" cellspacing="5" class="graph-container">
+
+              <tr>
+                <td class="your-activity-outer">
+                  <div class="your-activity-header">
+                    Points you collected this week
+                  </div>
+                  <div class="your-activity-badge">
+                    <%= weekly.user.pointsCollected %>
+                  </div>
+                  <div class="your-activity-footer">
+                    Class average: <strong><%= weekly.course.averages.pointsGenerated %></strong>
+                  </div>
+                </td>
+
+                <td class="your-activity-outer">
+                <% 
+                if (weekly.user.rank) { 
+                -%>
+                  <div class="your-activity-header">
+                    Your rank this week
+                  </div>
+                  <div class="your-activity-badge">
+                    <%= weekly.user.rank.thisWeek %>
+                  </div>
+                <% 
+                } 
+                -%>
+                </td>
+              </tr>
+
+              <tr>
+                <td>
+                  Total points so far: <strong><%= user.points %></strong>
+                <% 
+                if (course.engagementindex_url) { 
+                -%>
+                  <div class="outbound-button">
+                    <a href="<%= course.engagementindex_url %>">
+                      View leaderboard
+                    </a>
+                  </div>
+                <% 
+                } 
+                -%>
+                </td>
+              </tr>
+
+            </table>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <h4 class="section-header">
+              How others engaged with your contributions
```

Comment:
@paulkerschen UX question: Should we hide this section if user has had zero activity? Reporting zero comments/likes on zero contributions seems unnecessary.